### ISSUE TITLE
physics_raycast crashes the runner when an array is passed as the ids parameter

### ISSUE THREAD

**Issue by YYBartT** [YoYoGames/GameMaker-Bugs#5707]
### Description

The new function `physics_raycast` should accept an array of instance IDs and/or object assets, according to https://github.com/YoYoGames/GameMaker-Bugs/issues/2762#issuecomment-2069187096.

The runner silently crashes, however, when an array with valid items is passed as the `ids` argument, i.e.: 
```gml
var result = physics_raycast( xStart, yStart, xEnd, yEnd, obj_ball, true, 1 );   // Works
var result = physics_raycast( xStart, yStart, xEnd, yEnd, [obj_ball], true, 1 ); // Doesn't work
```

### Expected Change

The runner doesn't crash when arrays of objects and/or instances are provided

### Steps To Reproduce

1. Start GameMaker
2. Import the sample project in the comment here: https://github.com/YoYoGames/GameMaker-Bugs/issues/2762#issuecomment-2067697257
3. Run the project without changing anything. This runs without issues.
4. Change the `ids` parameter in the function call to `physics_raycast` made in the Draw event from `obj_ball` to `[obj_ball]` and run the project again. The runner silently crashes during the `physics_raycast` function call.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.400 (Betas)

### Which platform(s) are you seeing the problem on?

Windows

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by rwkay** [comment]
Fixed in 2024.6 - (Also had an issue on HTML5)
* Fixed problem when `max_fraction` was set to `undefined` and the default value would not be set.

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.600.0.568  Runtime v2024.600.0.595
